crates/pallet-history-seeding/src/lib.rs

```diff
@@ -0,0 +1,84 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::pallet_prelude::*;
+use frame_support::traits::{BuildGenesisConfig, Get};
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+
+#[cfg(test)]
+mod tests;
+pub mod weights;
+
+pub use pallet::*;
+
+#[frame_support::pallet]
+pub mod pallet {
+    use super::*;
```

Comment:
Please list the imported individual data structures explicitly and only those defined in the module, not imports.